lib/tasks/test_security_helper.rb

```diff
@@ -43,11 +46,18 @@ def self.brakeman(format: "human")
       :report_progress => $stderr.tty?,
       :use_prism       => true,
     }
-    if format == "json"
+    case format
+    when "json"
+      raise ArgumentError, "cannot pass --interactive-ignore with json output" if interactive_ignore
+
       options[:output_files] = [
         Rails.root.join("log/brakeman.json").to_s,
         Rails.root.join("log/brakeman.log").to_s
       ]
+    when "human"
+      options[:interactive_ignore] = true if interactive_ignore
+    else
+      raise ArgumentError, "Unknown format #{format.inspect}"
```

Comment:
Read this case statement reads aloud and tell me this is not what it says...

"The format is either json or human.  Therefore, JaSON isn't human."

🤣